cantor-s3/src/test/java/com/salesforce/cantor/s3/ObjectsOnS3Test.java

```diff
@@ -27,6 +27,6 @@ protected double getStoreMagnitude() {
     @Override
     protected Cantor getCantor() throws IOException {
         final AmazonS3 s3Client = S3Mock.getInstance().createS3Client("us-west-1");
-        return new CantorOnS3(s3Client);
+        return new CantorOnS3(s3Client, "warden-cantor");
```

Comment:
Yeah that makes sense, though what do you think of using `warden-cantor-dev`, since every new bucket requires SVS to go through the long process of requesting a new bucket? This way we can just use the dev environments bucket for testing.

Reply:
Open source Cantor should have nothing to do with Warden.